examples/react/playground/src/stories/SpaceCreatorCreating.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { SpaceCreatorCreating } from '@w3ui/react'
+import '@w3ui/react/src/styles/space-creator.css'
+
+export default {
+  title: 'w3ui/SpaceCreatorCreating',
+  component: SpaceCreatorCreating,
+  tags: ['autodocs'],
+  argTypes: {
+  },
+  parameters: {
+    backgrounds: {
+      default: 'dark'
+    }
+  }
+}
+
+export const Primary = {
+  args: {
+  }
+}
```

Comment:
same here, we can fix up the titles and exports to group states of the same logical components
```suggestion
export default {
  title: 'w3ui/SpaceCreator',
  component: SpaceCreatorCreating,
  tags: ['autodocs'],
  argTypes: {
  },
  parameters: {
    backgrounds: {
      default: 'dark'
    }
  }
}

export const Creating = {}
```

Reply:
ditto